packages/thumbprint-react/components/Calendar/index.tsx

```diff
@@ -122,9 +97,9 @@ export interface CalendarProps {
     /**
      * A react-day-picker modifier for greater control over disabled days. Past selection is
      * disabled by default.
-     * See: https://react-day-picker-v7.netlify.app/docs/matching-days
+     * See: https://react-day-picker.js.org/api/types/Matcher
      */
-    disabledDays?: Modifier | Modifier[] | null;
+    disabledDays?: Matcher | Matcher[] | null;
```

Comment:
My understanding is that this won't be a breaking change since only the name of the library's `disabledDays` prop has changed to `disabled`. I guess that means that `Modifier | Modifier[]` is equal to `Matcher | Matcher[]` in value?

Reply:
Yes, Both looks similar in type as mentioned below(Note: Name get changed little like RangeModifier to  DateRange, BeforeModifier to DateBefore):

V7: type Modifier = Date | RangeModifier | BeforeModifier | AfterModifier | BeforeAfterModifier | DaysOfWeekModifier | FunctionModifier | undefined;

V8: type Matcher = boolean | ((date: Date) => boolean) | Date | Date[] | DateRange | DateBefore | DateAfter | DateInterval | DayOfWeek;


Also, looks like new `boolean` matcher get added in v8.